lsp/src/service/filetypes/buildspec/strategy.ts

```diff
@@ -3,7 +3,7 @@ import { LanguageServiceStrategy } from '../../registry/types'
 
 import { BuildspecService } from './service'
 
-const BUILDSPEC_FILE_NAMES = ['build', 'buildspec'].map(name => [`${name}.yml`, `${name}.yaml`]).flat()
+export const BUILDSPEC_FILE_NAMES = ['build', 'buildspec'].map(name => [`${name}.yml`, `${name}.yaml`]).flat()
```

Comment:
I didn't see anywhere this is used outside this file. Is it necessary to export?